services/web/client/source/class/osparc/data/Resources.js

```diff
@@ -323,6 +323,23 @@ qx.Class.define("osparc.data.Resources", {
             url: statics.API + "/activity/status"
           }
         }
+      },
+
+      /*
+       * Test/Diagnonstic entrypoint
+       */
+      checkEP: {
```

Comment:
do you really used this? If so, POST needs a body to comply with openAPI

Reply:
We try to make all REST calls through this class and whether a body is required or not is not defined here.